docs/source/slice_a_pipeline.md

```diff
@@ -0,0 +1,32 @@
+# Slice a pipeline
+
+Slicing a pipeline refers to creating a subset of a pipeline's nodes, which can help in focusing on specific parts of the pipeline. There are two primary ways to achieve this in Kedro:
+
+1. **Programmatically with the Kedro CLI.** This method is ideal for those comfortable with command-line tools. Detailed steps on how to achieve this are available in the kedro documentation: [Slice a Pipeline](https://docs.kedro.org/en/0.19.5/nodes_and_pipelines/slice_a_pipeline.html).
```

Comment:
```suggestion
1. **Programmatically with the Kedro CLI.** This method is ideal for those comfortable with command-line tools. Detailed steps on how to achieve this are available in the Kedro documentation: [Slice a Pipeline](https://docs.kedro.org/en/latest/nodes_and_pipelines/slice_a_pipeline.html).
```

Should this point to `latest` in the docs? I don't remember what the normal practice is.

Reply:
I don't think we're being very consistent, but at least if we want to avoid hardcoding a specific version, we should use `/stable` instead of `/latest`